graphql-schemas/schema.yml

```diff
@@ -756,6 +757,24 @@
   - { name: user, type: User_v1, isRequired: true }
   - { name: requestId, type: string, isRequired: true }
 
+- name: AWSAccountSharingOption_v1
+  isInterface: true
+  interfaceResolve:
+    strategy: fieldMap
+    field: provider
+    fieldMap:
+      ami: AWSAccountSharingOptionAMI_v1
+  fields:
+  - { name: provider, type: string, isRequired: true }
+  - { name: account, type: AWSAccount_v1, isRequired: true }
+
+- name: AWSAccountSharingOptionAMI_v1
+  interface: AWSAccountSharingOption_v1
+  fields:
+  - { name: provider, type: string, isRequired: true }
+  - { name: account, type: AWSAccount_v1, isRequired: true }
+  - { name: regex, type: string, isRequired: true }
```

Comment:
please add a description. is this a regex of AMI IDs?

Reply:
there is a description in the json schema?

Reply:
ok see comment below